welcome/src/main/kotlin/edu/artic/welcome/WelcomeViewModel.kt

```diff
@@ -115,6 +118,14 @@ class WelcomeViewModel @Inject constructor(private val welcomePreferencesManager
         shouldPeekTourSummary.onNext(welcomePreferencesManager.shouldPeekTourSummary)
     }
 
+    fun updateData() {
+        val activeCardHolder = memberInfoPreferencesManager.activeCardHolder
+        activeCardHolder?.let { cardHolder ->
```

Comment:
Add comment saying `this only works because we don't support logout.`

Reply:
This doesn't have to go in this file - anywhere appropriate, really.

Reply:
I don't think adding that comment here makes sense. The Sign-In feature in the app is only to validate the membership information that the user provides. Here we are just pulling the validated active card-holder info from the file.

Reply:
Hmm. I suppose? Either way that's more appropriate for the README.